### ISSUE TITLE
cfvqa.py

### ISSUE THREAD

**Issue by GaomingL1n** [yuleiniu/cfvqa#20]
’logits_cfvqa‘ is TIE, ’z_qkv‘ is TE, ’logits_cfvqa‘ = ’z_qkv‘ - ’z_q’, but in the cfvqa.py,  ‘z_q’ is annotated as NIE, and later ’logits‘ is also annotated as NIE. Are the annotations mixed up? Or perhaps there's a problem with my understanding of the code？

**Comment by yangxuantao** [comment]
u r right .i think the code in cfvqa.py should be z_qkv(te) z_q(nde) logits_cfvqa(tie).I also don't understand why the" logit "is being labeled as NIE. The implementation of CFVQA in the paper does not mention the use of NIE
